src/components/onboarding/__stories__/artist_selector.story.tsx

```diff
@@ -0,0 +1,6 @@
+import { storiesOf } from "@storybook/react"
+import * as React from "react"
+
+import Artists from "../steps/artists";
+
+storiesOf("Onboarding").add("Artist Selector", () => <Artists onNextButtonPressed={null} />)
```

Comment:
I wasn't sure if it's right to do `onNextButtonPressed={null}`. Obviously, we don't any callbacks in the storybook, but passing in an empty function seems to make `yarn lint` angry.

Reply:
You can do e.g. `() => null`